contracts/Escrow.sol

```diff
@@ -171,14 +192,14 @@ contract Escrow {
             status == EscrowStatuses.Partial,
             "Escrow not in Pending or Partial status state"
         );
-        intermediateResultsUrl = _url;
-        intermediateResultsHash = _hash;
-        emit IntermediateStorage(_url, _hash);
+        recordingOracleResultsUrl = _url;
+        recordingOracleResultsHash = _hash;
+        emit RecordingOracleResults(_url, _hash);
```

Comment:
I like the rename

Reply:
cool. @gaieges , we are going to do this in a separate commit actually!